README.md

```diff
@@ -40,19 +40,17 @@ These features need to be enabled with the `next` option.
 
 - [Explicit resource management](https://github.com/tc39/proposal-explicit-resource-management)
 - [Source phase import](https://github.com/tc39/proposal-source-phase-imports)
-- [RegExp modifiers](https://github.com/tc39/proposal-regexp-modifiers) (See [RegExp support](#regexp-support))
 
 ## RegExp support
 
 Meriyah doesn't parse RegExp internal syntax, ESTree spec didn't require internal structure of RegExp. Meriyah
 does use JavaScript runtime to validate the RegExp literal. That means Meriyah's RegExp support is only as good
 as JavaScript runtime's RegExp support.
 
-As of August 2024, some latest RegExp features are not supported due to missing implementation in general
-JavaScript runtime.
+As of May 2025, some latest RegExp features requires Node.js>=24.
 
-- [RegExp modifiers](https://github.com/tc39/proposal-regexp-modifiers) (stage 3) is not supported
-- [RegExp duplicate named groups](https://github.com/tc39/proposal-duplicate-named-capturing-groups) is not supported
+- [RegExp modifiers](https://github.com/tc39/proposalegexp-modifiers)
+- [RegExp duplicate named groups](https://github.com/tc39/proposal-duplicate-named-capturing-groups)
```

Comment:
The line 52 url change is a mistake.

Reply:
Thanks, will fix when back to my laptop.